frontend/packages/kubevirt-plugin/src/utils/utils.ts

```diff
@@ -42,10 +45,10 @@ export const wrapWithProgress = (setProgress: (inProgress: boolean) => void) =>
 
 export const getVMLikeModelListPath = (isCreateTemplate: boolean, namespace: string) =>
   isCreateTemplate
-    ? `/k8s/ns/${namespace}/virtualization/templates`
-    : `/k8s/ns/${namespace}/virtualization`;
+    ? `/k8s/ns/${namespace}/${VIRTUALMACHINES_TEMPLATES_BASE_URL}`
+    : `/k8s/ns/${namespace}/${VIRTUALMACHINES_BASE_URL}`;
 
 export const getVMLikeModelDetailPath = (isTemplate: boolean, namespace: string, name: string) =>
```

Comment:
in the .json file you have plugins that redirect from a resource to the rendered page, so you know if it's template or not when you call who ever use this method, so the path can be constant in each call